src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -945,7 +947,7 @@ private String getBody(ScanResults.XIssue issue, ScanRequest request, String fil
         StringBuilder body = new StringBuilder();
         Optional.ofNullable(jiraProperties.getDescriptionPrefix())
                 .ifPresent(body::append);
-
+        Map<String, Object> additionalDetails = issue.getAdditionalDetails();
```

Comment:
This declaration should be moved closer to its usage.

Reply:
Fixed